.npmignore

```diff
@@ -1 +0,0 @@
-ros_foxglove_msgs
```

Comment:
unrelated to PR but this file is no longer needed as of https://github.com/foxglove/foxglove-sdk/pull/170